client/src/modules/blockEditor/actions.ts

```diff
@@ -62,11 +63,63 @@ export const exportSelection = () => {
     const { hoveredItem } = blockEditor;
 
     const block = blocks.blocks.find(b => b.id === hoveredItem.blockId);
+
     const uuid = uuidv1();
     if (block) {
+      const fragment = block.value.fragment;
+      const topLevelNodes = fragment.nodes.toJSON()[0].nodes;
+
+      const onlyOneNodeThatIsPointerExport = nodes => {
```

Comment:
Move the two functions and `spaceTextNode` somewhere closer to top-level so that they aren't redefined every time the export selection handler is called?

Reply:
https://github.com/oughtinc/mosaic/pull/145/commits/ab7abae57db354aba6cb665c102a4f813b9353e4